lib/core/Blockchain.ts

```diff
@@ -161,4 +164,33 @@ export default class Blockchain implements IBlockchain {
     console.info(`Refreshed blockchain time: ${responseBodyString}`);
     return responseBody;
   }
+
+  /**
+   * Fetches the normalized transaction fee used for proof-of-fee calculation, given the blockchain time.
+   * @param transactionNumber A valid Sidetree transaction number.
+   *
+   * @throws SidetreeError with ErrorCode.BlockchainTimeOutOfRange if the input transaction number is less
+   * than Sidetree genesis blockchain time or is later than the current blockchain time.
+   */
```

Comment:
Consider only have the interface method documentation at the interface definition, `IBlockchain.ts` in this case.